File: 프로그래머스/1/12901. 2016년/2016년.py
def solution(a, b):
    w=["THU","FRI","SAT","SUN","MON","TUE","WED"]
    m=[0,31,29,31,30,31,30,31,31,30,31,30,31]
    return w[(sum(m[:a])+b)%7]

File: 프로그래머스/1/12901. 2016년/2016년.js
function solution(a, b) {
    let w=["THU","FRI","SAT","SUN","MON","TUE","WED"]
    let m=[0,31,29,31,30,31,30,31,31,30,31,30,31]
    return w[(m.slice(0,a).reduce((add,item)=>add+item)+b)%7]
}